### PR TITLE
Fixes for CAS migration guide

### PR DESCRIPTION
**Pull request by rothgar**
**2. Description of changes:**
Needed to create an instance profile for karpenter

**3. How was this change tested?**
locally in a test cluster

**4. Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: *link to issue*
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/preview/getting-started/migrating-from-cas/scripts/step08-generate-chart.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 helm template --namespace karpenter \
     karpenter karpenter/karpenter \
-    --set aws.defaultInstanceProfile=KarpenterInstanceNodeRole \
+    --set aws.defaultInstanceProfile=KarpenterInstanceProfile \
     --set clusterEndpoint="${OIDC_ENDPOINT}" \
```
Comment (njtran) [inline website/content/en/preview/getting-started/migrating-from-cas/scripts/step08-generate-chart.sh:4]: Can you check if this is right @rothgar ?

https://github.com/aws/karpenter/pull/1642#discussion_r849969424

Comment (rothgar) [reply website/content/en/preview/getting-started/migrating-from-cas/scripts/step08-generate-chart.sh:4]: Good catch. https://github.com/aws/karpenter/blob/main/pkg/utils/options/options.go#L37=

**Comment by njtran** [comment]
Can you make the changes to the v0.8.2 docs as well? Since we made a release after the initial commit, it's in two places now.

File: website/content/en/preview/getting-started/migrating-from-cas/_index.md [hunk 2]
```diff
@@ -112,14 +122,14 @@ Replace the nodegroup value with your `${NODEGROUP}`
```
Comment (njtran) [inline website/content/en/preview/getting-started/migrating-from-cas/_index.md:122]: Could you change L110 to say `- ${NODEGROUP}` to be more specific? this way it fails if they forget to change it. 

Same with L145

File: website/content/en/preview/getting-started/migrating-from-cas/_index.md [hunk 3]
```diff
@@ -150,15 +160,15 @@ Modify the `ng-123456` value to match your `$NODEGROUP`.
 Now that karpenter is running we can disable the cluster autoscaler.
 To do that we will scale the number of replicas to zero.
 
-{{% script file="./content/en/{VERSION}/getting-started/migrating-from-cas/scripts/step11-scale-cas.sh" language="bash" %}}
+{{% script file="./content/en/{VERSION}/getting-started/migrating-from-cas/scripts/step12-scale-cas.sh" language="bash" %}}
 
 To get rid of the instances that were added from the node group we can scale our nodegroup down to a minimum size to support Karpenter and other critical services.
 We suggest a minimum of 2 nodes for the node group.
 
 > Note: If your workloads do not have [pod disruption budgets](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) set
```
Comment (njtran) [inline website/content/en/preview/getting-started/migrating-from-cas/_index.md:168]: nit: 
```suggestion
> Note: If your workloads do not have [pod disruption budgets](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) set,
```

File: website/content/en/preview/getting-started/migrating-from-cas/_index.md [hunk 4]
```diff
@@ -150,15 +160,15 @@ Modify the `ng-123456` value to match your `$NODEGROUP`.
...
 > the following command **will cause workloads to be unavailable**
```
Comment (njtran) [inline website/content/en/preview/getting-started/migrating-from-cas/_index.md:169]: nit: 
```suggestion
> the following command **will cause workloads to be unavailable.**
```

File: website/content/en/preview/getting-started/migrating-from-cas/_index.md [hunk 5]
```diff
@@ -150,15 +160,15 @@ Modify the `ng-123456` value to match your `$NODEGROUP`.
...
 
-{{% script file="./content/en/{VERSION}/getting-started/migrating-from-cas/scripts/step12-scale-ng.sh" language="bash" %}}
+{{% script file="./content/en/{VERSION}/getting-started/migrating-from-cas/scripts/step13-scale-ng.sh" language="bash" %}}
 
 If you have a lot of nodes or workloads you may want to slowly step down your node groups by a few instances at a time.
```
Comment (njtran) [inline website/content/en/preview/getting-started/migrating-from-cas/_index.md:173]: nit: 
```suggestion
If you have a lot of nodes or workloads you may want to slowly scale down your node groups by a few instances at a time.
```

**Review by njtran — APPROVED** [review]
LGTM!

**Pull request merged**